tests/pages/package-search.js

```diff
@@ -78,7 +76,7 @@ import { isRootPresent } from './helpers';
     isSelected: computed(function () {
       return this.isSelectedText === 'Selected';
     }),
-    isActive: is('[class*="is-selected"]'),
+    isActive: is(this.$root, '[class*="is-selected"]'),
```

Comment:
Because of the way instance initializers work, I'm pretty sure `this` refers to the instance of `PackageSearchPage`. So It's weird that this works. The old way should be the right way, since if you omit the selector it defaults to the current instance's `$root`.

If the old way doesn't work and it for-sure is not an actual failure, then this is most definitely a bug in `@bigtest/interactor`.

Same goes for the other few times this happens (I noticed it's always with `is`)

Reply:
`isActive: is('[class*="is-selected"]')` fails with `Error: unable to find "[class*="is-selected"]"`

Reply:
🐛 

Fix incoming

Reply:
Fix released. Try `@bigtest/interactor` `0.4.2`

Reply:
New `interactor` release put in to fix `is()`.